src/models/worldStateModel.ts

```
@@ -0,0 +1,613 @@
+        Season: Number,
+        Phase: Number,
+        Params: String,
+        ActiveChallenges: [ActiveChallengeSchema],
+        UsedChallenges: [String]
+    },
+    { _id: false }
+);
+
+SeasonInfoSchema.set("toJSON", {
+    transform(_document, returnedObject) {
+        delete returnedObject.UsedChallenges;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)